package.json

```
@@ -1,7 +1,8 @@
 {
   "license": "UNLICENSED",
   "devDependencies": {
-    "@semantic-release/commit-analyzer": "^10.0.1",
+    "@getoutreach/semantic-release-circleci-orb": "^1.1.10",
+    "@semantic-release/commit-analyzer": "^11.0.0",
```

suggestion: probably want to avoid upgrading semantic-release dependencies
```suggestion
    "@semantic-release/commit-analyzer": "^10.0.1",
```

---

I was actually changing it back to what it was.

---

https://github.com/getoutreach/devbase/pull/729/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

---

`stencil-base` says it's `^10.0.1`: https://github.com/getoutreach/stencil-base/blob/e6b648200a77a82ac665ba8213dd9e6bde7aeaa5/templates/package.json.tpl#L4